{
  "dataType": "CVE_RECORD",
  "containers": {
    "cna": {
      "title": "Glance-store: glance store access key logged in debug log level",
      "credits": [
        {
          "lang": "en",
          "value": "Red Hat would like to thank Lujie (ICT) for reporting this issue."
        }
      ],
      "metrics": [
        {
          "other": {
            "type": "Red Hat severity rating",
            "content": {
              "value": "Moderate",
              "namespace": "https://access.redhat.com/security/updates/classification/"
            }
          }
        },
        {
          "format": "CVSS",
          "cvssV3_1": {
            "scope": "UNCHANGED",
            "version": "3.1",
            "baseScore": 5.5,
            "attackVector": "LOCAL",
            "baseSeverity": "MEDIUM",
            "vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N",
            "integrityImpact": "NONE",
            "userInteraction": "NONE",
            "attackComplexity": "LOW",
            "availabilityImpact": "NONE",
            "privilegesRequired": "LOW",
            "confidentialityImpact": "HIGH"
          }
        }
      ],
      "affected": [
        {
          "cpes": [
            "cpe:/a:redhat:openstack:17.1::el9"
          ],
          "vendor": "Red Hat",
          "product": "Red Hat OpenStack Platform 17.1 for RHEL 9",
          "versions": [
            {
              "status": "unaffected",
              "version": "0:2.5.1-17.1.20230621023901.el9ost",
              "lessThan": "*",
              "versionType": "rpm"
            }
          ],
          "packageName": "python-glance-store",
          "collectionURL": "https://access.redhat.com/downloads/content/package-browser/",
          "defaultStatus": "affected"
        },
        {
          "cpes": [
            "cpe:/a:redhat:openstack:16.1"
          ],
          "vendor": "Red Hat",
          "product": "Red Hat OpenStack Platform 16.1",
          "packageName": "python-glance-store",
          "collectionURL": "https://access.redhat.com/downloads/content/package-browser/",
          "defaultStatus": "unknown"
        },
        {
          "cpes": [
            "cpe:/a:redhat:openstack:16.2"
          ],
          "vendor": "Red Hat",
          "product": "Red Hat OpenStack Platform 16.2",
          "packageName": "python-glance-store",
          "collectionURL": "https://access.redhat.com/downloads/content/package-browser/",
          "defaultStatus": "unknown"
        }
      ],
      "timeline": [
        {
          "lang": "en",
          "time": "2024-01-17T00:00:00+00:00",
          "value": "Reported to Red Hat."
        },
        {
          "lang": "en",
          "time": "2024-01-17T00:00:00+00:00",
          "value": "Made public."
        }
      ],
      "datePublic": "2024-01-17T00:00:00+00:00",
      "references": [
        {
          "url": "https://access.redhat.com/errata/RHSA-2024:2732",
          "name": "RHSA-2024:2732",
          "tags": [
            "vendor-advisory",
            "x_refsource_REDHAT"
          ]
        },
        {
          "url": "https://access.redhat.com/security/cve/CVE-2024-1141",
          "tags": [
            "vdb-entry",
            "x_refsource_REDHAT"
          ]
        },
        {
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2258836",
          "name": "RHBZ#2258836",
          "tags": [
            "issue-tracking",
            "x_refsource_REDHAT"
          ]
        }
      ],
      "workarounds": [
        {
          "lang": "en",
          "value": "Avoid leaving the DEBUG log level enabled in critical environments."
        }
      ],
      "descriptions": [
        {
          "lang": "en",
          "value": "A vulnerability was found in python-glance-store. The issue occurs when the package logs the access_key for the glance-store when the DEBUG log level is enabled."
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "CWE",
              "cweId": "CWE-779",
              "description": "Logging of Excessive Data"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
        "shortName": "redhat",
        "dateUpdated": "2024-05-22T21:45:05.759Z"
      },
      "x_redhatCweChain": "CWE-779: Logging of Excessive Data"
    },
    "adp": [
      {
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2024-1141",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "partial"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-02-01T20:50:59.471008Z"
              }
            }
          }
        ],
        "providerMetadata": {
          "shortName": "CISA-ADP",
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "dateUpdated": "2024-04-29T17:57:33.644Z"
        }
      }
    ]
  },
  "cveMetadata": {
    "cveId": "CVE-2024-1141",
    "state": "PUBLISHED",
    "dateUpdated": "2024-05-22T21:45:05.759Z",
    "dateReserved": "2024-02-01T00:47:57.686Z",
    "assignerOrgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
    "datePublished": "2024-02-01T14:21:37.758Z",
    "assignerShortName": "redhat"
  },
  "dataVersion": "5.1"
}